src/components/User.tsx

```diff
@@ -0,0 +1,43 @@
+import { sessionApi } from '../components/sessionApi';
+import { useState } from 'react';
+
+class User {
+  isLoggedIn = () => this.get('isLoggedIn') === 'true';
+
+  set = (key: string, value: string) => localStorage.setItem(key, value);
+  get = (key: string) => this.getLocalStorage(key);
```

Comment:
why u use localStorage ? localStorage can be editted by user. This is security hole